index.html

```diff
@@ -52,38 +55,87 @@ <h2 class="h2-header-info">2021.07.30(FRI)~31(SAT)</h2>
             <div class="progam-disc">
                 <img src="images/lecture.png" alt="Lecture" />
                 <h3>Lecture</h3>
-                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt ea enim repellat ad sit ipsum</p>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
             </div>
             <div class="progam-disc">
                 <img src="images/exhibition.PNG" alt="Lecture" />
                 <h3>Exihibition</h3>
-                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt ea enim repellat ad sit ipsum</p>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
             </div>
             <div class="progam-disc">
                 <img src="images/forum.PNG" alt="Lecture" />
                 <h3>Forum</h3>
-                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt ea enim repellat ad sit ipsum</p>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
             </div>
             <div class="progam-disc">
                 <img src="images/workshop.PNG" alt="Lecture" />
                 <h3>Workshop</h3>
-                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt ea enim repellat ad sit ipsum</p>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
             </div>
             <div class="progam-disc">
                 <img src="images/ignite.PNG" alt="Lecture" />
                 <h3>Ignite</h3>
-                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Nesciunt ea enim repellat ad sit ipsum</p>
+                <p>Lorem ipsum dolor sit amet consectetur adipisicing elit.</p>
             </div>
 
         </div>
         <p id="wholeProgram">SEE THE WHOLE PROGRAM</p>
-    </div>
+        <button type="button" id="programButton">SEE THE WHOLE PROGRAM</button>
+    </section>
+
+    <section class="feature_speakers">
+        <div class="feature-container">
+          <h2>Featured Speakers</h2>
+          </div>
+        <div class="underline"></div>
+        <ul class="feature-list"></ul>
+    </section>
+
+    <section class="partners">
+        <h3>Partner</h3>
+        <div class="underline"></div>
+
+        <div class="partnersImage">
+            <div class="partnerName">
+                <img src="./images/partner.PNG" alt="partner">
+                <p>Bank of Baku</p>
+            </div>
+            <div class="partnerName">
+                <img src="./images/partner.PNG" alt="partner">
+                <p>SOCAR XXXX</p>
+            </div>
+            <div class="partnerName">
+                <img src="./images/partner.PNG" alt="partner">
+                <p>MOOVER XXX</p>
+            </div>
+            <div class="partnerName">
+                <img src="./images/partner.PNG" alt="partner">
+                <p>Uni Bank NM</p>
+            </div>
```

Comment:
* [OPTIONAL]
* You can still improve this section on mobile. YOURS : 
![image](https://user-images.githubusercontent.com/52497246/125173876-8fa3ad00-e1b9-11eb-8f57-104f6eac07e7.png)
ORIGINAL : 
![image](https://user-images.githubusercontent.com/52497246/125173888-a3e7aa00-e1b9-11eb-90fe-4c1cd6d36460.png)

* I'm sorry this point didn't show up in the earlier review that is why this change is marked OPTIONAL.